Makefile

```diff
@@ -9,8 +9,8 @@ LDFLAGS = -O2
 
 all: wake-on-arp
 
-debug: LDFLAGS = -g
-debug: CFLAGS += -g
```

Comment:
How did your old build support the DEBUG field? I was a bit confused why the `ifdef DEBUG` blocks didn't get compiled after finding out that this field need to be manually defined.